packages/calcite-components/src/components/list/list.tsx

```diff
@@ -160,6 +160,16 @@ export class List
    */
   @Prop({ reflect: true }) loading = false;
 
+  /**
+   * Specifies the fields to match against when filtering. If not set, all fields will be matched (label, description, metadata, value).
```

Comment:
Sorry, late to the party here - is "fields" an obvious term here for this? "If not set, all fields will be matched (label, description, metadata, value)"

Really it's matching supplied property values? Feel free to disregard but "fields" doesn't immediately convey what will be matched to me.

Reply:
We already called the property `matchFields` on `filter.tsx` but we can change before we release if we have a beter name. `matchPropertyValues` or `matchKeyValues`? I think its really a key. Could also be `matchKeys`?

@jcfranco

Reply:
I kinda like `filterProps`. // the properties to use for filtering? @macandcheese? @jcfranco

Reply:
We will likely need to update the changelog at release to modify matchFields to filterProps on the filter component.